app/src/main/java/ch/epfl/sweng/swenggolf/offer/Category.java

```diff
@@ -11,4 +16,44 @@
     public static Category getDefault() {
         return Category.OTHER;
     }
```

Comment:
Just as a side note: I've removed this method in my PR since it is not used anywhere. Do you have any objection against the removal?

Reply:
Isn't it used in `ListOfferActivityTest.java`? 
https://github.com/SwengGolfTeam/Sweng_Golf/blob/fa789475dc4feef8bfb3cb9691e5a8fdffd15136/app/src/androidTest/java/ch/epfl/sweng/swenggolf/ListOfferActivityTest.java#L176

And in `Offer.java` ?
https://github.com/SwengGolfTeam/Sweng_Golf/blob/1bacb0429f626219d3221ba0ef9a0cd1a2a3ed56/app/src/main/java/ch/epfl/sweng/swenggolf/offer/Offer.java#L70
https://github.com/SwengGolfTeam/Sweng_Golf/blob/1bacb0429f626219d3221ba0ef9a0cd1a2a3ed56/app/src/main/java/ch/epfl/sweng/swenggolf/offer/Offer.java#L93

Reply:
You're absolutely right, I must have been drunk.